### DISCUSSION TITLE
Lizenz nach Festplattenverlust reaktivieren

### DISCUSSION THREAD

**Discussion by uwearzt** [brutella/hkknx-public#197] [Q&A]
Hi, wollte gerade eine komplette Neueinrichtung nach einem Festplattencrash machen. Leider erhalte ich folgende Fehlermeldung:

<img width="771" alt="Screenshot 2022-07-16 at 14 40 17" src="https://user-images.githubusercontent.com/105293/179355355-57c7a157-1411-44a2-88e0-24a206252dcf.png">

Kann ich die Lizenz reaktivieren?

Danke, Uwe

**Comment by brutella** [comment]
Bitte sende eine E-Mail an hkknx@hochgatterer.me mit Bekanntgabe deiner Lizenz-E-Mail Adresse.
